task/goreleaser/0.1/goreleaser.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: goreleaser
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: golang, release-automation, package
+    tekton.dev/displayName: "GoReleaser"
+spec:
+  description: >-
+    GoReleaser builds Go binaries for several platforms.
+
+    It creates a GitHub release and then pushes a Homebrew formula to a tap repository.
+  workspaces:
+    - name: source
+      mountPath: /workspace/src/$(params.package)
+      description: >-
+        The workspace containing the Go source code
+        which needs to be released.
+  params:
+    - name: package
+      description: base package to build in
+    - name: github-token-secret
+      description: name of the secret holding the github-token
+      default: bot-token-github
+    - name: github-token-secret-key
+      description: name of the secret key holding the github-token
+      default: bot-token
+    - name: flags
+      description: flags to use for the test command
```

Comment:
I think this command is wrong (it doesnt aply only for the test command), I believe you can just say 

```
flags to pass to `goreleaser release`
```